Created the FAQ and Donate Page

Lucasha11 opened the pull request:
Created both the FAQ and Donate pages

Lucasha11 committed:
Created the FAQ and Donate Page

ShawnAviles commented:
@Lucasha11 pls put photo of the pages that were added - would make reviewing PR a lot easier so we know what we are looking for

ShawnAviles commented:
There is only a jpg file added in this PR... no markdown so I assume it's not done or you may have not committed the pages yet

Lucasha11 committed:
Created the FAQ and Donate Pages

Lucasha11 committed:
Created the FAQ and Donate Pages

Lucasha11 committed:
Created the FAQ and Donate Pages

Lucasha11 commented:
FAQ Page:
<img width="1440" alt="Screenshot 2024-12-11 at 1 03 08 PM" src="https://github.com/user-attachments/assets/4ab2928c-a876-4a17-be1e-166be1710296" />
<img width="1440" alt="Screenshot 2024-12-11 at 1 03 30 PM" src="https://github.com/user-attachments/assets/3baab9b3-3424-4350-a5e0-21a4ed9907a5" />
<img width="1440" alt="Screenshot 2024-12-11 at 1 03 44 PM" src="https://github.com/user-attachments/assets/a716c72a-1357-4623-be2a-4bae404a1113" />
<img width="1440" alt="Screenshot 2024-12-11 at 1 03 57 PM" src="https://github.com/user-attachments/assets/756c892a-c39d-4a23-8a48-323fdcd6662c" />

Donate Page: 
<img width="1440" alt="Screenshot 2024-12-11 at 1 05 23 PM" src="https://github.com/user-attachments/assets/5b98840e-db26-48f2-92cc-ac705949415e" />
<img width="1440" alt="Screenshot 2024-12-11 at 1 05 31 PM" src="https://github.com/user-attachments/assets/3d49146a-7f66-4dd4-81d7-9c1c312328e1" />

Lucasha11 committed:
Populated the Accordion blocks in the FAQ page

Lucasha11 committed:
Populated the Accordion blocks in the FAQ page

ShawnAviles committed:
Update hero banner with button and darken background (#98)

Update hero banner component to includes
- option for action button that links to another page
- toggle to darken background with transparent layer

Example on `http://localhost:3000/admin/index.html#/~/example-hero`

![Screenshot 2024-12-11 at 2 03
34 AM](https://github.com/user-attachments/assets/22f6d487-d7dc-4631-9861-ae660600538c)

![Screenshot 2024-12-11 at 2 04
15 AM](https://github.com/user-attachments/assets/79ada15e-a7fb-4b84-bbd2-f8b54ba2e451)

![Screenshot 2024-12-11 at 2 04
28 AM](https://github.com/user-attachments/assets/1c2a80e2-e17a-431c-91ca-e61198589300)

Tina Options:

![Screenshot 2024-12-11 at 2 04
53 AM](https://github.com/user-attachments/assets/593a28bb-7606-4cce-a814-0ed4066b9a92)

ShawnAviles committed:
Update hero banner with button and darken background (#98)

Update hero banner component to includes
- option for action button that links to another page
- toggle to darken background with transparent layer

Example on `http://localhost:3000/admin/index.html#/~/example-hero`

![Screenshot 2024-12-11 at 2 03
34 AM](https://github.com/user-attachments/assets/22f6d487-d7dc-4631-9861-ae660600538c)

![Screenshot 2024-12-11 at 2 04
15 AM](https://github.com/user-attachments/assets/79ada15e-a7fb-4b84-bbd2-f8b54ba2e451)

![Screenshot 2024-12-11 at 2 04
28 AM](https://github.com/user-attachments/assets/1c2a80e2-e17a-431c-91ca-e61198589300)

Tina Options:

![Screenshot 2024-12-11 at 2 04
53 AM](https://github.com/user-attachments/assets/593a28bb-7606-4cce-a814-0ed4066b9a92)

ShawnAviles committed:
Update hero banner with button and darken background (#98)

Update hero banner component to includes
- option for action button that links to another page
- toggle to darken background with transparent layer

Example on `http://localhost:3000/admin/index.html#/~/example-hero`

![Screenshot 2024-12-11 at 2 03
34 AM](https://github.com/user-attachments/assets/22f6d487-d7dc-4631-9861-ae660600538c)

![Screenshot 2024-12-11 at 2 04
15 AM](https://github.com/user-attachments/assets/79ada15e-a7fb-4b84-bbd2-f8b54ba2e451)

![Screenshot 2024-12-11 at 2 04
28 AM](https://github.com/user-attachments/assets/1c2a80e2-e17a-431c-91ca-e61198589300)

Tina Options:

![Screenshot 2024-12-11 at 2 04
53 AM](https://github.com/user-attachments/assets/593a28bb-7606-4cce-a814-0ed4066b9a92)

johanngg7 committed:
make footer responsive (#61)

Co-authored-by: Justin Ferber <ferberjustin5@gmail.com>
Co-authored-by: Justin Ferber <119906373+j-ferber@users.noreply.github.com>

ShawnAviles committed:
HOT FIX - Rich Text Column Layout by accident (#104)

# Description

Fix wrong styling of rich-text

Thank you for noticing @SevenThanh

Lucasha11 committed:
Changed name of FAQ page

Lucasha11 committed:
Changed name of FAQ page

miguel-merlin committed:
Rich Text positioning (#112)

The size of the button inside the `RichText` component spanned all the
container. If fixed it by letting the user provide a positioning option
in the `RichText` component.
<img width="1624" alt="image"
src="https://github.com/user-attachments/assets/3cb21dab-315a-4a42-b50d-1ba18450fd59"
/>

---------

Co-authored-by: Shawn Aviles <shawnaviles26@gmail.com>

miguel-merlin committed:
Rich Text positioning (#112)

The size of the button inside the `RichText` component spanned all the
container. If fixed it by letting the user provide a positioning option
in the `RichText` component.
<img width="1624" alt="image"
src="https://github.com/user-attachments/assets/3cb21dab-315a-4a42-b50d-1ba18450fd59"
/>

---------

Co-authored-by: Shawn Aviles <shawnaviles26@gmail.com>

miguel-merlin committed:
Rich Text positioning (#112)

The size of the button inside the `RichText` component spanned all the
container. If fixed it by letting the user provide a positioning option
in the `RichText` component.
<img width="1624" alt="image"
src="https://github.com/user-attachments/assets/3cb21dab-315a-4a42-b50d-1ba18450fd59"
/>

---------

Co-authored-by: Shawn Aviles <shawnaviles26@gmail.com>

miguel-merlin committed:
Team Gallery title fix (#111)

## Description
Removed the roleName to be rendered. Instead the user will now add the
roleName in the Markdown Box. This was needed to match some of the
designs for the intern and volunteers sections of the Gallery.

<img width="1624" alt="image"
src="https://github.com/user-attachments/assets/5975d334-76eb-4d7e-a156-5ad00d328e71"
/>

miguel-merlin committed:
Fixed width Image Text Button Component (#110)

## Description
Fixed width of Image Text Button Component to match provided image in
issue

<img width="1624" alt="image"
src="https://github.com/user-attachments/assets/1cd0699b-c096-41a3-909e-63b2dbbb2bc2"
/>

Solves #102

miguel-merlin committed:
Background color Two column and Rich Text (#109)

## Description
Added white bg color to Two Column and Rich Text component

miguel-merlin committed:
Linear Gradient to Here bg image (#113)

## Description
I added a color filter to the background image in the Hero component. 
<img width="1624" alt="image"
src="https://github.com/user-attachments/assets/47bc2fa9-d181-41cd-be68-4b666869f47f"
/>
<img width="1624" alt="image"
src="https://github.com/user-attachments/assets/f3ce80bb-a864-4cc9-852b-a945c6125b13"
/>

miguel-merlin committed:
Linear Gradient to Here bg image (#113)

## Description
I added a color filter to the background image in the Hero component. 
<img width="1624" alt="image"
src="https://github.com/user-attachments/assets/47bc2fa9-d181-41cd-be68-4b666869f47f"
/>
<img width="1624" alt="image"
src="https://github.com/user-attachments/assets/f3ce80bb-a864-4cc9-852b-a945c6125b13"
/>

miguel-merlin committed:
Linear Gradient to Here bg image (#113)

## Description
I added a color filter to the background image in the Hero component. 
<img width="1624" alt="image"
src="https://github.com/user-attachments/assets/47bc2fa9-d181-41cd-be68-4b666869f47f"
/>
<img width="1624" alt="image"
src="https://github.com/user-attachments/assets/f3ce80bb-a864-4cc9-852b-a945c6125b13"
/>

Lucasha11 committed:
Deleted extra FAQ page and edited address

Lucasha11 committed:
..

Lucasha11 commented:
[Donate page](http://localhost:3000/donate)
[FAQs Page](http://localhost:3000/FAQs)

ShawnAviles commented on public/FAQ/FAQs cover.jpg:
Should there be duplicates of these images saved?

ShawnAviles commented on content/pages/Donate.mdx:
```diff
@@ -0,0 +1,42 @@
+        # <ColoredText text="DONATE" color="White" />
+      position: Left
+    _template: hero
+  - richTextLeft: "<YoutubeEmbed url=\"https://www.youtube.com/watch?reload=9&v=l_sm8Q1r3d8\" />\n\nYour donation matters! Here’s the impact you can make:\n\n\U0001F49A $25 = 21 more people supported\n\U0001F49A $50 = 42 more people supported\n\U0001F49A $100 = 84 more people supported\n\U0001F49A $250 = 210 more people supported\n\U0001F49A $500 = 420 more people supported\n\U0001F49A $1,000 = 8 ...
```
I believe the youtube video id is incorrect. doesn't play anything

<img width="1482" alt="Screenshot 2024-12-17 at 11 33 56 PM" src="https://github.com/user-attachments/assets/622fd59c-db3f-4ae9-b08f-c8057312be98" />

ShawnAviles reviewed (changes requested):
There were a lot of changes made to components in the PR, was that intentional? or is that some issue from not pulling from main and other commits got added to this? @miguel-merlin @Lucasha11

ShawnAviles commented:
Also the routes are all caps, everywhere else they are all lowercase.

i.e. `/Donate` should be `/donate` and `/FAQs` should be `/faqs` for consistency

Lucasha11 committed:
changed embedded youtube video and made donate address lowercase

Lucasha11 committed:
renamed pages and fixed embedded youtube video

Lucasha11 committed:
Changed FAQ address to 'faqs'

Lucasha11 committed:
edited photo and did nothing else

Lucasha11 committed:
Fixed the Rich Text merge conflict

Lucasha11 committed:
Fix the tina-lock.json conflict

Lucasha11 committed:
Merge branch 'main' into FAQ_and_Donate

Lucasha11 committed:
Merge branch 'main' into FAQ_and_Donate

j-ferber committed:
Changed headers and file names

j-ferber committed:
formatting

j-ferber committed:
Merge branch 'main' into FAQ_and_Donate

j-ferber reviewed (approved):
LGTM
